docs/configuration/configuration.md

```diff
@@ -125,6 +126,8 @@ An example [oauth2-proxy.cfg]({{ site.gitweb }}/contrib/oauth2-proxy.cfg.example
 | `--tls-key-file` | string | path to private key file | |
 | `--upstream` | string \| list | the http url(s) of the upstream endpoint, file:// paths for static files or `static://<status_code>` for static response. Routing is based on the path | |
 | `--user-id-claim` | string | which claim contains the user ID | \["email"\] |
+| `--groups-claim` | string | which claim contains the user groups | `"groups"` |
+| `--required-group` | string \| list | required groups which the user is required to be a member of | `[]` |
```

Comment:
We're starting to have a bazillion `*-group` flags that all specify groups for authorization logic. I don't know the right answer to this:

Do we consolidate any providers that support groups under a single flag?

Or do we name this in a way that it is specific to OIDC analogous to `google-groups` or `gitlab-groups` (or less so `github-teams`)? IE `oidc-group`

In general the OIDC only flags are more loosy-goosy in the naming and documentation and don't always indicate they are OIDC provider only, though.

@JoelSpeed @steakunderscore thoughts? I'm curious how this fits into the future vision for structured configs.

Reply:
Spoke with @JoelSpeed - we do want to consolidate these various similar group flags eventually.  But I think we can do that in another PR scoped to that consolidation. You can continue as planned